core/src/main/resources/reference.conf

```diff
@@ -1020,8 +1020,9 @@ datastax-java-driver {
     # the package com.datastax.oss.driver.internal.core.addresstranslation.
     #
     # The driver provides the following implementations out of the box:
-    # - PassThroughAddressTranslator: returns all addresses unchanged
+    # - PassThroughAddressTranslator: returns all addresses unchanged.
     # - FixedHostNameAddressTranslator: translates all addresses to a specific hostname.
+    # - SubnetAddressTranslator: translates addresses to hostname based on the subnet matches.
```

Comment:
Like `advertised-hostname` below, we should also add a commented out example of configuring `subnet-addresses` for `SubnetAddressTranslator`